debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -695,18 +744,17 @@ public static EventProcessingFailureHandlingMode parse(String value) {
                                                            + "'never' to specify the connector should never run a snapshot and that upon first startup the connector should read from the beginning of the binlog. "
                                                            + "The 'never' mode should be used with care, and only when the binlog is known to contain all history.");
 
-    public static final Field SNAPSHOT_MINIMAL_LOCKING = Field.create("snapshot.minimal.locks")
```

Comment:
We aim at not breaking backwards compatability as far as possible, so I'd prefer a two-step phase out process: first deprecate it, and then remove it in a future version. So we'd keep it for now, and it'd equate to snapshot locking mode being set to "minimal". If both options, this one and the new one, are given, the new one would take precedence. If the old option is given, a warning will be printed out.